tools/mitohifi/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: mitohifi
+owner: bgruening
+description:  Circularises, cuts and annotates the mitogenome
+homepage_url: https://github.com/marcelauliano/MitoHiFi 
+long_description: |
+  Circularises, cuts and annotates the mitogenome from contigs assembled with PacBio HiFi reads and software such as HiCanu or Hifiasm.
+remote_repository_url: https://https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi
+type: unrestricted
+categories:
+- TODO
```

Comment:
Can you choose a category for toolshed.g2.bx.psu.edu/

Maybe "Genome annotation"